source/frontend/src/hooks/repositories/useComposedProperties.ts

```diff
@@ -157,31 +157,46 @@ export const useComposedProperties = ({
     id,
     retrievedPid,
     retrievedPin,
-    getLtsaWrapper,
-    getPropertyWrapper,
-    getPropertyAssociationsWrapper,
-    findByWrapper,
-    getPropertyWfsWrapper,
-    getSummaryWrapper,
+    getLtsaWrapper.response,
+    getPropertyWrapper.response,
+    getPropertyAssociationsWrapper.response,
+    findByWrapper.response,
+    getPropertyWfsWrapper.response,
+    getSummaryWrapper.response,
   ]);
 
-  return {
-    id: id,
-    pid: pid?.toString() ?? retrievedPid,
-    pin: retrievedPin,
-    composedProperty: composedProperty,
-    ltsaWrapper: getLtsaWrapper,
-    apiWrapper: getPropertyWrapper,
-    propertyAssociationWrapper: getPropertyAssociationsWrapper,
-    parcelMapWrapper: findByWrapper,
-    geoserverWrapper: getPropertyWfsWrapper,
-    bcAssessmentWrapper: getSummaryWrapper,
-    composedLoading:
-      getLtsaWrapper?.loading ||
-      getPropertyWrapper?.loading ||
-      getPropertyAssociationsWrapper?.loading ||
-      findByWrapper?.loading ||
-      getPropertyWfsWrapper?.loading ||
-      getSummaryWrapper?.loading,
-  };
+  return useMemo(
```

Comment:
interesting , the memo isn't required in the other two locations but is here? any idea why?